irohad/consensus/yac/storage/impl/yac_vote_storage.cpp

```diff
@@ -69,6 +69,11 @@ namespace iroha {
               return storage->insert(state) |
                          [this, &round](
                              auto &&insert_outcome) -> boost::optional<Answer> {
+
+                if (round > last_round_) {
+                  last_round_ = round;
+                }
```

Comment:
`last_round_` is `boost::optional`:
```
                last_round_ = std::max(last_round_.value_or(round), round);
```

Reply:
Optional is required because in the first round there is no committed state yet.